infra/openshift-manifests/minio-webhook-source/ksvc.yaml

```diff
@@ -11,7 +11,7 @@ spec:
       containers:
         - image: quay.io/kevent-mesh/ai-demo-minio-webhook-source:main
           ports:
-            - containerPort: 5000
+            - containerPort: 8080
```

Comment:
8080 is the default, so we can drop it

Reply:
hm, I create a new task for this, to drop it in all services